### ISSUE TITLE
[quick deploy] For first time deploy or clean deploy, is it better to clean storage path (/datastorage)?

### ISSUE THREAD

**Issue by YanjieGao** [Microsoft/pai#968]
For first time deploy or clean deploy senearios, (not upgrading service).
When cluster once deployed PAI
I execute:
1 paictl k8s-clean
2 paictl deploy
as /datastorage data is not clean, meet this problem [as worker and master current/VERSION is different]
![image](https://user-images.githubusercontent.com/5576848/43119565-758a2f9a-8f49-11e8-83a9-8f4595a847f9.png)

3 then i reconfig path to a clean folder. The above problem not occurs

**Comment by ydye** [comment]
As you described, I don't think it's a first time deployment

**Comment by YanjieGao** [comment]
change title to redeploy. just a suggestion.  This problem different solution have different advantage and drawback.
Keep data could keep hdfs previous data. But may let user for quick try pai occurs hdfs error.

**Comment by ydye** [comment]
Yes, it's better to perform a clean operator. But notice that this operator will clean pai relative data on the disk.
